package.json

```diff
@@ -2,7 +2,7 @@
     "name": "mindaro",
     "displayName": "Bridge to Kubernetes",
     "description": "Rapid Kubernetes development for teams",
-    "version": "2.0.120230124",
+    "version": "2.0.120230216",
```

Comment:
Hi Elena seems like testing team is having issues with this release, let's hold on for more investigation and proceed once it is clear. Thank you.